Gemfile.lock

```diff
@@ -31,7 +31,7 @@ GEM
       racc (~> 1.4)
     nokogiri (1.14.2-x86_64-linux)
       racc (~> 1.4)
-    octokit (6.0.1)
+    octokit (6.1.0)
```

Comment:
Our CI or our Dependabot config feels a little broken that it's bumping all deps, not just the one mentioned in the PR...